src/components/FilterToDos.js

```diff
@@ -0,0 +1,49 @@
+import "./FilterToDos.css";
+
+export default function FilterToDos({ filterType, setFilterType }) {
+  function handleAllFilterClick() {
```

Comment:
The logic of these `handleAllFilterClick`and the other 2 (for Done and ToDo) should be done in the `App` component